Automattic/wp-calypso
# Split document head selectors

This is a small PR that splits document head selectors into individual modules, in preparation for some modularisation work. There are no code changes other than a few bits of code moving around.

#### Changes proposed in this Pull Request

* Split document head selectors into individual modules
* Import document head selectors directly from their individual modules

#### Testing instructions

Since there are no real code changes, the unit tests and a clean build should be enough to ensure that everything works properly. Feel free to also smoke-test the live branch; this handles the document head, so any breakage would be very obvious on every page.
